src/database/Database.cpp

```diff
@@ -366,6 +366,23 @@ Database::initialize()
     // normally you do not want to touch this section as
     // schema updates are done in applySchemaUpgrade
 
+    if (isSqlite())
+    {
+        mSession << "PRAGMA journal_mode = OFF";
```

Comment:
why not just guarding the whole block with `if (isSqlite() && mApp.getConfig().DATABASE.value != "sqlite3://:memory:")` ?

Reply:
No particular reason. I can do that.